<launch>
    <arg name="debug" default="false" />
    <node name="motion_controller_adjuster_node" pkg="motion_controller" type="motion_controller_adjuster_node" respawn="false" output="screen" >
        <param name="transport_hint" value="raw" />
        <param name="debug" value="$(arg debug)" />
    </node>
</launch>